english/explore-outside/journey-to-flaming-tower/home/home.md

```diff
@@ -1 +1,7 @@
-While walking home you stumbled upon a field of strawberries. Not any ordinary strawberries, talking ones
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.